bootstrap/helpers/notifications.php

```
@@ -13,6 +14,35 @@ function is_transactional_emails_enabled(): bool
+}
+
+function mail_from_address(object $settings): Address
+{
+    $identity = mail_from_identity($settings);
+
+    return new Address($identity['address'], $identity['name']);
+}
+
+function mail_from_formatted(object $settings): string
+{
+    return (string) mail_from_address($settings);
```

PHPStan (2.2.7)

[warning] Cannot cast Symfony\Component\Mime\Address to string.


(cast.string)